content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,382 @@
+Interscambio)` with the buyer *themselves* as **recipient** (using their own :guilabel:`Codice
+Destinatario`). That way, they can deduct their VAT. The **self-billed** invoice must be issued and
+recorded in the VAT sales register within **15 days** from receipt.
+
+Customer invoices
+=================
+
+The :guilabel:`Fattura PA` feature is enabled by default, and can be found under
+:menuselection:`Accounting --> Configuration --> Journals --> Customer Invoices (journal) -->
+Advanced Settings (tab) --> Electronic Data Interchange`. Once :doc:`created and confirmed
+<../../getting_started/process_overview/customer_invoice>`, the e-Invoice is generated and sent. You
+can check the current status of an invoice by the :guilabel:`Electronic invoicing` field *(cf.
+screenshot below)*. The XML file can be found in the **chatter** of the invoice.
+
+.. warning::
+   e-Invoices are only sent to the government if the :abbr:`EDI (Electronic Document Invoicing)` is
+   set to :guilabel:`Official` mode.
+
+.. image:: italy/italy-processing.png
    :align: center
    :alt: Electronic invoicing status (waiting for confirmation)
 
+.. _italy/down-payments:
+
+Down payments
+-------------
+
+**Down payments** invoices are imported/exported with a different :guilabel:`Typo Documento` code
+(`TD02`) than regular invoices. The import creates a regular vendor bill.
+
+Odoo exports moves as `TD02` if the following conditions are met:
+
+- is an invoice;
+- All invoice lines are related to **sales order lines** that have the flag `is_downpayment` set as
+  `True`.
+
+Simplified invoices and credit notes
+------------------------------------
+
+A simplified invoice can be used to certify **domestic transactions** under **400 EUR** (VAT
+included). Its status is the same as a regular invoice, but with fewer information requirements.
+
+For a **simplified** invoice to be established, it must include:
+
+- :guilabel:`Customer Invoice` reference: **unique** numbering sequence with **no gaps**;
+- :guilabel:`Invoice Date`: issue **date** of the invoice;
+- :guilabel:`Company Info`: the **seller**'s full credentials (VAT/TIN number, name, full address)
+  under :menuselection:`General Settings --> Companies (section)`;
+- :guilabel:`VAT`: the **buyer**'s VAT/TIN number (on their profile card);
+- :guilabel:`Total`: the total **amount** (VAT included) of the invoice.
+
+In the :abbr:`EDI (Electronic Data Interchange)`, Odoo exports invoices as simplified if:
+
+- It is a **domestic** transaction (i.e., the partner is from Italy);
+- The buyer's data is **insufficient** for a regular invoice;
+- The **required fields** for a regular invoice (address, ZIP code, city, country) are provided;
+- The total amount VAT included is **less** than **400 EUR**.
+
+Invoices can have different :guilabel:`Tipo Documento`, depending on the move type:
+
+- `TD07`: Simplified invoice;
+- `TD08`: Simplified credit note.
+
```

Comment:
```suggestion

```
No debit note in V14, right ? Couldn't find any

Reply:
There is, but it's very discouraged flow. 
We actually currently import TD09 kind-ed documents as normal invoices in the code in v14.
There also is the `account_debit_note` module in v14 (which I don't know much about).